advance_ai_agents/finance_service_agent/routes/agentRoutes.py

```
@@ -0,0 +1,177 @@
+                    "parameters": [
+                        {"name": "format", "type": "string", "description": "Response format (html or json)"}
+                    ],
+                    "example_query": "",
+                    "example_response": json.dumps(error_response, indent=2),
+                    "current_year": current_year
+                }
+            )
+            
+        return JSONResponse(content=error_response)
+
+@router.get("/chat", response_class=HTMLResponse)
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)